libpod/container.go

```diff
@@ -628,6 +628,15 @@ func (c *Container) RuntimeName() string {
 
 // Hostname gets the container's hostname
 func (c *Container) Hostname() string {
+	if c.config.UTSNsCtr != "" {
+		utsNsCtr, err := c.runtime.GetContainer(c.config.UTSNsCtr)
+		if err != nil {
+			// should we return an error here?
+			logrus.Errorf("unable to lookup uts namespace for container %s: %v", c.ID(), err)
+			return ""
```

Comment:
Nit: I think it would be better to change the signature to return error here and let caller deal with it.

Reply:
Likely, however this function is used in many places right now and I didn't want to update all callers. Since this is somewhat unrelated to this change. I don't think we will ever hit this error.
@mheon PTAL